code/lab/labv2.cpp

```diff
@@ -0,0 +1,26 @@
+#include "lab/labv2.h"
+#include "lab/labv2_internal.h"
+#include "lab/manager/lab_manager.h"
+#include "gamesequence/gamesequence.h"
+
+LabManager* LMGR;
+
+void lab_init() {
+	gr_set_clear_color(0, 0, 0);
+
+	LMGR = new LabManager();
+
+	Game_mode |= GM_LAB;
```

Comment:
Is this reset anywhere?

Reply:
Yes. When the user leaves the Lab, LabManager::close is called which removes the GM_LAB flag again.

Reply:
Ah, ok. A bit confusing to do the "set" part here and the "unset" in a different location but nothing critical.